src/DependencyInjection/Configuration.js

```diff
@@ -24,6 +24,11 @@ class Configuration extends implementationOf(ConfigurationInterface) {
                     .isRequired()
                     .info('The tsconfig.json location')
                 .end()
+                .arrayNode('webpack_extra_configs')
+                    .scalarPrototype().end()
+                    .defaultValue(['%kernel.project_dir%/extra-webpack.config.js'])
```

Comment:
Empty array as default value. Do not assume any extra configuration.

Reply:
:+1